types/global.d.ts

```diff
@@ -78,6 +80,7 @@ declare namespace NodeJS {
     location: Location;
     navigator: Navigator;
     newrelic: NewRelic.Browser;
+    ga: Function | undefined;
```

Comment:
Are there actual types we can use here?

Reply:
yes actually, https://www.npmjs.com/package/@types/google.analytics